fancyshowcaseview/src/main/java/me/toptas/fancyshowcase/FancyShowCaseView.kt

```diff
@@ -185,9 +185,6 @@ class FancyShowCaseView @JvmOverloads constructor(context: Context, attrs: Attri
         })
     }
 
-    /**
-     * Starts enter animation of FancyShowCaseView
-     */ @TargetApi(Build.VERSION_CODES.LOLLIPOP)
```

Comment:
`@TargetApi` annotation should be there

Reply:
Apologies, didn't spot that. Added back in and just made a fresh commit.